documentation/modules/proc-designating-strimzi-administrators.adoc

```diff
@@ -11,14 +11,14 @@
 
 .Procedure
 
-. Create the `strimzi-admin` cluster role in Kubernetes.
+. Create the `strimzi-view` and `strimzi-admin` cluster roles in Kubernetes.
 +
 Use `kubectl apply`:
 [source,shell,subs=+quotes]
 kubectl apply -f install/strimzi-admin
 
-. Assign the `strimzi-admin` `ClusterRole` to one or more existing users in the Kubernetes cluster.
+. If needed, assign the roles to one or more existing users in the Kubernetes cluster.
```

Comment:
_If needed_? If we're designating strimzi admins, we would need to do this step, no?

Reply:
Because of the aggregation, even without this step the rights will be given to everyone with one of the default roles. So this step is now not mandatory.

Reply:
Maybe it could be clearer. _If needed, assign the roles that provide access rights to users that require them._

Reply:
Ok, fixed. Thanks